lib/ansible/modules/cloud/amazon/cloudwatchlogs_log_group.py

```diff
@@ -259,12 +259,15 @@ def main():
                                                retention=module.params['retention'],
                                                module=module)
         elif found_log_group:
-            if module.params['retention'] != found_log_group['retentionInDays']:
```

Comment:
A simpler fix would be 

```
if module.params['retention'] != found_log_group.get('retentionInDays'):
```

wouldn't it?

I'm not at all convinced of the need to reformat the call to `input_retention_policy` - PEP8 does not express a preference either way: https://www.python.org/dev/peps/pep-0008/#id17.

Reply:
Our coding style is "Whatever is enforced by CI"

Reply:
@willthames i've just noticed this comment. Firstly sorry for not replying sooner. I get your point about the reformatting. tbh though its still v.valid. seems a bit silly PR's like this simple one, fixing an actual bug, get stuck on someones personal preference of styling?